.doctrine-project.json

```diff
@@ -10,11 +10,23 @@
             "slug": "latest",
             "upcoming": true
         },
+        {
+            "name": "0.3",
+            "branchName": "0.3.x",
+            "slug": "0.3",
+            "upcoming": true
```

Comment:
There's already a 0.3 tag, so I don't know if it's an upcoming or a current version. The default branch is still 0.2.x.

Reply:
The default branch should be changed to 0.3.x, and I lacked the necessary permissions for that. Can you please do it?

Reply:
Looks like I haven't the permissions either.